python/openassetio/constants.py

```diff
@@ -65,6 +65,10 @@
 ## @}
 
 
+## The maximum integer value of a valid management policy bitfield
+kMaxManagementPolicyValue = 31
```

Comment:
[question] How was this number derived?  Naively,
```python
>>> 1 | 3 | 8 | 16
27
```
am I missing something?  If I add in the `7` then I get `31`, but the `7` is for `kStateMask`, which presumably isn't a state in itself? Or is it that we want to reserve 3 bits for future state?